src/main/java/servlets/VendorServlet.java

```diff
@@ -33,12 +45,75 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     JsonConverter converter = new JsonConverter();
     String configMap = converter.getConfigMap();
-    response.setContentType("text/html;");
+    response.setContentType(CONTENT_TYPE);
     response.getWriter().println(configMap);
   }
 
-  @Override
-  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // DELETE method --> deletes vendor configuration directory
+  /** Delete the desired Vendor from the fileset and spreadsheets if posible.*/
```

Comment:
nit: "possible"

Reply:
Maybe explain why it wouldn't be possible? This comment is a bit confusing to me

Reply:
The only reason it wouldn't delete a file is if the user inputted an invalid vendorID. Once Vincent's PR is pushed, we shouldn't need to worry about that happening.